<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chessboard_ai</class>
 <widget class="QWidget" name="Chessboard_ai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="re_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>再来一局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hui_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="bai_num">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="hei_num">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hei_piece">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>黑棋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bai_piece">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>白棋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
